src/main/kotlin/org/jetbrains/intellij/IntelliJPluginConstants.kt

```diff
@@ -43,6 +43,9 @@ object IntelliJPluginConstants {
     const val RELEASE_TYPE_NIGHTLY = "nightly"
     const val RELEASE_TYPE_RELEASES = "releases"
 
+    const val BUILD_FEATURE_CHECK_GRADLE_VERSION = "org.jetbrains.intellij.buildFeature.checkGradleVersion"
+    const val BUILD_FEATURE_CHECK_GRADLE_INTELLIJ_PLUGIN_VERSION = "org.jetbrains.intellij.buildFeature.checkGradleIntellijPluginVersion"
```

Comment:
why do we re-define the property name here? re-use enum name or add `configPropertyName` property to enum and externalize prefix

Reply:
Well, enum was a leftover. But constructing consts like that – I don't like it anymore and I'll get back to enums now. 🤷‍♂️